packages/react-components/package.json

```diff
@@ -31,9 +31,11 @@
     "core-js": "3.29.1",
     "date-fns": "2.29.3",
     "date-fns-tz": "1.3.8",
+    "dompurify": "^2.2.9",
```

Comment:
Does markdown still leave us open to attack? I kind of feel doing this client side might be a mistake as what you see in Contentful might not reflect the hub

Reply:
we're using `rehype-sanitize` instead of dompurify (also for rich text). I agree that it's worth considering. But I think that discussion should be separated from this, and we should take care of that sanitization on both markdown and rich text in the BE.